src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecorator.java

```diff
@@ -431,4 +437,37 @@ public int getExitCode() {
             return i;
         }
     }
+    
+	static class FilterOutExitCodeOutputStream extends OutputStream {
```

Comment:
can we move it to a new class `org.csanchez.jenkins.plugins.kubernetes.pipeline.exec. FilterOutExitCodeOutputStream ` with 

```
@Restricted(NoExternalUse.class)
```